docs/user_manual/type_data.md

```diff
@@ -352,22 +352,30 @@ that are intended to be used together. Index sequences should all be unique with
 
 Some index families are dual-indexed. This means that one or two indices can be added to each library. This increases
 the possibilities for unique sequences. Some dual-indexed families have an index 1 and index 2 that are always matched
-together. These are referred to as "unique dual index" families within MISO. In other index families, any index 1 may
-be matched with any index 2.
+together. These are referred to as "unique dual index" families within MISO. For unique dual index families, the name
+of the matching index 1 and index 2 in MISO must be identical in order for them to be automatically selected together.
+In other index families, any index 1 may be matched with any index 2.
 
 Some Illumina sequencers, including the NextSeq, sequence the index 2 as the
 reverse complement of other sequencers. MISO has an option to set the
 sequencing platform to use the normal index in the interface and provide the
 reverse complement in sample sheets and though the Pinery data export
 interface.
 
+In some index families such as 10X Genomics kits, multiple sequences are associated with a single index. These are
+referred to as multi-sequence indices in MISO.
```

Comment:
Could the Fake/Demultiplexing sequence thing get a few sentences here?

Reply:
Added